docs/source/models/concept_models/vivarium_nutrition_optimization/kids/concept_model.rst

```diff
@@ -469,9 +469,40 @@ Requested test runs:
 Subnational Approach
 ~~~~~~~~~~~~~~~~~~~~
 
-.. todo::
-
-  Add general information here on the subnational approach planned. Include: data on which location IDs to use and the percent of the population in each ID. Information on intializing sims with a country and subnational location. What data to pull subnationally vs nationally.
+In order to include SQ-LNS targeting by location, we are switching to 
+use a subnational approach for most data in Wave 3. However, rather 
+than model all subnational locations separately, simulants 
+will just be assigned to a subnational location within their primary 
+location, and have input data pulled for the subnational location instead. 
+Unless otherwise specified in the model request table below, the data outputs 
+do not need to stratified by subnational location.
+
+
+Initializing Simulant's Locations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Simulants will be obtained from the pregnancy sim, the same as in prior 
+waves. These simulants will already have a country location. The pregnancy 
+simulation is only run at the national level.
+
+When these simulants are loaded into the child simulation, they will be 
+assigned a subnational location within their country. Here is the data 
```

Comment:
I think it's worth thinking about whether it is best to assign sublocation at birth versus at six months of age...

I think if we are not planning on modeling the first 6 months of life at the subnational level AND assign sublocation at birth, then we won't get the sublocation-specific mortality rates correct in the first six months of life, so our population distribution of sublocations will be off by the time the simulated population ages to six months (basically we will overestimate the population size of the high-mortality sublocations and vise versa because we won't have modeled corresponding mortality rates but rather just the overall average rate). So I think if we want to assign sublocation at birth, then we should also model everything, including LBWSG, at the subnational level.

Otherwise, we could use the average of the population weights between the 1-5 and 6-11 month age groups to initialize a subpopulation starting at six months of age? Maybe they are not that different so it won't matter, but I think worth looking into!

Reply:
Marking this as a topic for the MNCH RT tomorrow. I have compared the birth to 6 months ages groups to facilitate our conversation.